generic-ci.sh

```diff
@@ -39,11 +39,11 @@ case $1 in
     ;;
 
   verify-official)
-    jq ".\"post-processors\"[0] |= map(select(.\"type\" != \"vagrant-cloud\"))" vagrant.json | ./packer validate -var "iso_url=https://downloads.archlinux.de/iso/$(date +'%Y.%m').01/archlinux-$(date +'%Y.%m').01-x86_64.iso" -var "iso_checksum_url=https://downloads.archlinux.de/iso/$(date +'%Y.%m').01/sha1sums.txt" -
+    jq ".\"post-processors\"[0] |= map(select(.\"type\" != \"vagrant-cloud\"))" vagrant.json | ./packer validate -var "iso_checksum_url=https://mirror.pkgbuild.com/iso/latest/sha1sums.txt" -
```

Comment:
I quite don't understand why you don't need to set the iso_url here.

Reply:
From what I understand only the URL format is checked, not the reachability (there is no download as part of the validation process for iso_url) however the "good" formatting of the checksum file is done, and the resulting download name (given by iso_url) should match what is inside the checksum file. I will wait for your merge of the other PR for the cloud documentation and re-submit this PR. for information it will look like this in the end (it's a good thing I had to re-check):
<code>./packer validate -var "iso_checksum_url=https://mirror.pkgbuild.com/iso/latest/sha1sums.txt" -except=vagrant-cloud vagrant.json</code>
and
<code>./packer validate local.json</code>

So no more jq needed (which is not shipped by default with arch).